python/turbine_models/tests/stateless_llama_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import turbine_models.custom_models.stateless_llama as llama
+import unittest
+import os
+import pytest
+
+from typing import Literal
+
+
+def test_export(quantization: Literal["int4", None], precision: Literal["f16", "f32"]):
+    llama.export_transformer_model(
+        hf_model_name="llSourcell/medllama2_7b",
```

Comment:
I think this model is actually slightly different than the meta version. Might be better to use "Trelis/Llama-2-7b-chat-hf-function-calling-v2" or a secret hf token for the actual meta model we use